tests/accessor/accessor_api_image_common.h

```diff
@@ -945,14 +936,56 @@ class image_accessor_api_sampled_r {
     return (v000 + v001 + v010 + v011 + v100 + v101 + v110 + v111) / 8;
   }
 
+  template <typename coordTag>
   T get_expected_value(image_id_t<dim, target> idx) const {
     const bool useLinear =
         m_sampler.filtering_mode == sycl::filtering_mode::linear;
 
     if (!useLinear) {
       return get_expected_value_nearest(idx);
     }
-    return get_expected_value_linear(idx);
+    constexpr bool useUpper =
+        std::is_same<coordTag, acc_coord_tag::use_normalized_upper>::value;
```

Comment:
Oh no! You have started to use `std::is_same_v` everywhere instead. Go on! :-)

Reply:
Fixed by 69a4cf9461f7b71c16a475d7335cb88c86869b63 :-)